app/models/destination.rb

```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
 
 class Destination < ActiveRecord::Base
-  validates :url, presence: true
+  enum service_type: {growth_spaces: "growth_spaces", adobe_campaigns: "adobe_campaigns"}
+
+  validates :url, :service_type, presence: true
+  validates :route_id, presence: true, if: :growth_spaces?
```

Comment:
The presence of `route_id` should always be required (not just for `growth_spaces`).

Reply:
@hlbraddock what value should it contain for adobe records?

Reply:
Some number (though the actual column data type is `character varying`) just like for growth spaces. It's the way the client specifies the service desired. All services will therefore have route id's.

Reply:
@hlbraddock actually clients specify the service they want using a `destination_id` as a parameter of the request:

https://github.com/CruGlobal/mobile-content-api/blob/master/app/controllers/follow_ups_controller.rb#L13

`route_id` is a growth_space related parameter:

https://www.growthspaces.org/api/docs#subscribers